draft-ietf-quic-datagram.md

```diff
@@ -160,11 +160,11 @@ if the max_datagram_frame_size transport parameter is not present.
 # Datagram Frame Types
 
 DATAGRAM frames are used to transmit application data in an unreliable manner.
-The DATAGRAM frame type takes the form 0b0011000X (or the values 0x30 and 0x31).
-The least significant bit of the DATAGRAM frame type is the LEN bit (0x01). It
-indicates that there is a Length field present. If this bit is set to 0, the
-Length field is absent and the Datagram Data field extends to the end of the
-packet. If this bit is set to 1, the Length field is present.
+The Type field in the DATAGRAM frame takes the form 0b0011000X (or the values
+0x30 and 0x31). The least significant bit of the Type field in the DATAGRAM frame
```

Comment:
Could expand the parenthetical here if it would help clarify the `X`, but seems fine either way.